src/validation.cpp

```diff
@@ -934,6 +937,11 @@ static bool AcceptToMemoryPoolWorker(const CChainParams& chainparams, CTxMemPool
             }
         }
 
+        if (test_accept) {
+            *test_accept = true;
+            return false;
```

Comment:
Why not use the return value, instead of modifying a pointer?

Reply:
To keep the interface clear. It wasn't accepted to the memory pool, which is what the return value of ATMP denotes.

Reply:
It was accepted, just not added... Acceptance is exactly what you're testing here. IMO the interface is less clear the way you have it.

Reply:
Makes sense. Fixed